src/app/pages/datasets/components/dataset-management/dataset-management.component.html

```diff
@@ -119,14 +119,13 @@
 
     <div
       class="details-container"
-      [class.details-container-mobile]="showMobileDetails"
+      [class.details-container-mobile]="showMobileDetails$ | async"
```

Comment:
please follow the ordering rules 
**ixDetailsHeight** should be higher than **[class.details-container-mobile]**